lib/spack/docs/index.rst

```diff
@@ -65,6 +65,7 @@ or refer to the full manual below.
    repositories
    command_index
    package_list
+   gpg
```

Comment:
Agreed, but I didn't know where else to put it. `basic_usage` didn't seem right. Maybe `repositories` since it is intended for signing binary packages on mirrors?